spec/mailers/previews/alert_mailer_preview.rb

```diff
@@ -73,4 +85,44 @@ def example_no_med_training_for_course
               article: example_article,
               course: example_course)
   end
+
+  public
+
+  def create_example_survey_response_alert
+    answer = create_answer
+    details =
+      {
+        question: answer.question.question_text,
+        answer: answer.answer_text,
+        followup: answer.follow_up_answer_text,
+        source: SurveyResponseAlertManager.source(answer)
```

Comment:
This probably needs to become `SurveyResponseAlertManager.new.source(answer)`.

Reply:
Oups oups oups ...
Corrected